client/consul_test.go

```diff
@@ -2,106 +2,158 @@ package client
 
 import (
 	"context"
-	"errors"
+	"encoding/json"
+	"net/http"
 	"testing"
+	"time"
 
 	"github.com/hashicorp/consul-terraform-sync/logging"
+	"github.com/hashicorp/consul-terraform-sync/retry"
+	"github.com/hashicorp/consul-terraform-sync/testutils"
+	consulapi "github.com/hashicorp/consul/api"
 	"github.com/stretchr/testify/assert"
 )
 
-func Test_isConsulEnterprise(t *testing.T) {
+func Test_GetLicense_API_Failure(t *testing.T) {
 	t.Parallel()
+	path := "/v1/operator/license"
+	ctx := logging.WithContext(context.Background(), logging.NewNullLogger())
```

Comment:
Addressed in [315ac6a](https://github.com/hashicorp/consul-terraform-sync/pull/801/commits/315ac6a12ef5d854474c011085f5718c757a7fe5)

Reply:
Sorry, I forgot to mention, I think it may be possible to switch to `ctx := context.Background()` for the other test funcs as well: Test_GetLicense, Test_IsEnterprise_API_Failure, Test_IsEnterprise

Reply:
Yep, I'll do that and make a new PR